Automattic/jetpack
# Sync: PSR-4 the wp-super-cache sync module

This PR updates the wp-super-cache sync module to be loaded via PSR-4.

**Note:** Ideally this module should be part of wp-super-cache
Parent Issue: https://github.com/Automattic/jetpack/issues/12712

#### Changes proposed in this Pull Request:
* Update the WP-Super-Cache module 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Part of the Jetpack DNA project - p1HpG7-70O-p2

#### Testing instructions:
* Checkout the branch.
* Enable `WP_DEBUG_LOG`.
* Install WP-Super-Cache alongside with Jetpack.
* Play with the site, trigger a full sync.
* Verify sync works well and you have no errors logged.
* Verify tests pass and CI is green.

#### Proposed changelog entry for your changes:
* Sync: PSR-4 the WP-Super-Cache sync module
